src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -65,3 +68,12 @@
   <cd-usage-bar [totalBytes]="row.stats.stat_bytes"
                 [usedBytes]="row.stats.stat_bytes_used"></cd-usage-bar>
 </ng-template>
+</tab>
+
+<tab heading="Overall Performance" i18n-heading>
```

Comment:
List every tag attribute in its own line.

Reply:
ack